doc/source/getting_started/_notebook_stubs/fourier_transform.ipynb

```diff
@@ -0,0 +1,257 @@
+    "    \\int_{\\mathbb{R}^d} f(x)\\ e^{-i \\langle x, \\xi\\rangle} \\, \\mathrm{d}x\n",
+    "$$\n",
+    "\n",
+    "and can be applied to, e.g., functions in $L^2(\\mathbb{R}^d)$. ODL offers a way to use the discretization of this transform as an operator, thus making it available for all thinkable purposes in numerical methods."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import odl\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we define a discretization of the complex space $L^2(\\Omega, \\mathbb{C})$ for a rectangular domain $\\Omega \\subset \\mathbb{R}^2$. We choose $\\Omega = [-2, 2] \\times [0, 1]$ and square pixels:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "l2_discr = odl.uniform_discr([-2, 0], [2, 1], nsamples=(400, 100), dtype=complex)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "l2_discr.domain"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "l2_discr.cell_sides  # Side lengths of the pixels"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we createt the Fourier transform operator on this domain. Its range is inferred by default but can be specified explicitly. For the computation of the trigonometric sums (DFT), the [Numpy backend built on FFTPACK](http://docs.scipy.org/doc/numpy/reference/routines.fft.html) is used by default. ODL also supports the significantly faster [pyFFTW](https://pypi.python.org/pypi/pyFFTW) backend (requires installation of that package)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "fourier_trafo = odl.trafos.FourierTransform(l2_discr)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now the FT is available as an operator. We can check on which domain the transformed functions are defined:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "fourier_trafo.range.domain  # The definition domain of functions in the range of the transform"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The side lenghts of the rectangle are $2\\pi / s$, where $s = 0.01$ is the pixel side length of the real space grid. Note that the rectangle is slightly shifted into the negative direction (this can be controlled with a parameter)."
```

Comment:
Formatting of this is *ugly* (the raw code, not the output). Anyway, what parameter?